README.md

```diff
@@ -3,6 +3,15 @@
 ![logo-solid-bright](https://raw.githubusercontent.com/codingteam/logo/master/logo-solid-bright.png)
 ![logo-solid-dark](https://raw.githubusercontent.com/codingteam/logo/master/logo-solid-dark.png)
 
+## Installation
+
+If you want to use the logo in your project, one way of doing that is to install
+the logo through npm:
+
+```console
+$ npm install codingteam/logo#1.0.0 
```

Comment:
Yes this command installs the logo locally (into `node_modules`). Yes it still requires me to add the `1.0.0` tag to the repository. I'll do that after the merge. We'll definitely check the syntax in scope of https://github.com/codingteam/codingteam.org.ru/pull/52 (we have this working already, but it's set to download `feature/npm` branch instead of tag).